assets/images/fast-forward.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 192.689 192.689">
```

Comment:
I got this svg code from here: http://www.flaticon.com/free-icon/fast-forward_130903